docs/source/developers/guide/step_by_step/styling.rst

```diff
@@ -0,0 +1,69 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+
+.. SCOPE OF THIS SECTION
+.. This section should include extra description to the
+.. language-specific documentation. Possible topics to
+.. add: How to run one test, test file or all the tests
+.. together and why is it good to do that.
+.. What if the unconnected tests start failing? etc.
+
+
+.. _styling:
+
+**********
+Styling 😎
+**********
+
+Each language in the Apache Arrow project follows its own style guides.
+
+In this section we will connect to the existing documentation
```

Comment:
"connect" is a teensy bit unclear; perhaps instead something like "provide links to" or similar?

Reply:
Link is better, agree. Will do.